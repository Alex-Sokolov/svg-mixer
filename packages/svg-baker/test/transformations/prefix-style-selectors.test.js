const { prefixStyleSelectors } = require('svg-baker/lib/transformations');

const t = utils.testPlugin(prefixStyleSelectors);

it('should prefix all <style> tag selectors', async () => {
  expect(await t(
    '.qwe',
    '<svg><defs><style>.a {} .b .c {}</style></defs></svg>',
  )).toMatchSnapshot();
});
