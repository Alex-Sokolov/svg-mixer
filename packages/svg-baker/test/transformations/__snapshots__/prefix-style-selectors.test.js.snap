// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Users/kisenka/projects/svg-baker/packages/svg-baker/test/transformations/prefix-style-selectors.test.js 1`] = `"<svg><defs><style>.qwe .a {} .qwe .b .c {}</style></defs></svg>"`;

exports[`should prefix all <style> tag selectors 1`] = `"<svg><defs><style>.qwe .a {} .qwe .b .c {}</style></defs></svg>"`;
